test/unittest/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ endif()
 
 if(GTEST_FOUND)
     enable_testing()
+    find_package(Threads REQUIRED)
```

Comment:
Did you test this on El Capitan or Sierra?  In other sections we're adding pthread a little differently in order to not break OSX (where pthread is not available).  ex: https://github.com/apache/incubator-mxnet/blob/master/CMakeLists.txt#L226

Reply:
It works for me.

Reply:
What does this do?